### PR TITLE
Passenger capacity align with siri

### PR DESCRIPTION
**Pull request by ue71603**
fixes: https://github.com/NeTEx-CEN/NeTEx/issues/565

### CODE REVIEW

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 1]
```diff
@@ -737,6 +737,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>The number of wheelchairs places on vehicles of the type. For a requirement this is the minimum needed.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="PramPlaceCapacity" type="xsd:nonNegativeInteger" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:740]: Everything is NumberOfPassegers, lets do it for these two to?
```suggestion
			<xsd:element name="PramPlaceCapacity" type="NumberOfPassengers" minOccurs="0">
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 2]
```diff
@@ -737,6 +737,16 @@ Rail transport, Roads and Road transport
...
+				<xsd:annotation>
+					<xsd:documentation>The number of places on the vehicle that are suitable for prams.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BicycleRackCapacity" type="xsd:nonNegativeInteger" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:745]:
```suggestion
			<xsd:element name="BicycleRackCapacity" type="NumberOfPassengers" minOccurs="0">
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 3]
```diff
@@ -737,6 +737,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>The number of wheelchairs places on vehicles of the type. For a requirement this is the minimum needed.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="PramPlaceCapacity" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The number of places on the vehicle that are suitable for prams.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BicycleRackCapacity" type="NumberOfPassengers" minOccurs="0">
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:745]: Bicycle places are not limited to racks, e.g., there are systems with straps or just free space for any kind of luggage. Here a proposal with a different name. Note that the example would have to be adapted as well...
```suggestion
			<xsd:element name="BicycleCapacity" type="NumberOfPassengers" minOccurs="0">
```

**Review by trurlurl — CHANGES_REQUESTED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 3 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:745]: @trurlurl I agree but the element has that name in SIRI. So if we change it here, I would need to put a PR in SIRI As well.  @skinkie @Aurige your opinion?

Comment (trurlurl) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:745]: Yes, I realized it too late that we can't just change the element name. But can we change the definition / description - or is it important to only count places related to racks? If not, the description should help ensuring the interpretation we want.

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 4]
```diff
@@ -737,6 +737,16 @@ Rail transport, Roads and Road transport
...
+				<xsd:annotation>
+					<xsd:documentation>The number of bicycle racks on the vehicle.</xsd:documentation>
```
Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:747]:
```suggestion
					<xsd:documentation>The number of bicycle racks on the vehicle. Also non-rack places should be counted here. The name might change in future to BicycleCapacity.</xsd:documentation>
```

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 3 above)
Comment (Aurige) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:745]: I'd rather align with SIRI

**Review by skinkie — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**